src/dxvk/dxvk_presenter.cpp

```diff
@@ -68,11 +78,13 @@ namespace dxvk {
   VkResult Presenter::presentImage(
           VkPresentModeKHR  mode,
           uint64_t          frameId) {
+    std::lock_guard<dxvk::mutex> lock(m_lowLatencyMutex);
```

Comment:
QueuePresent is also allowed to block, although this might be less of a concern if the driver is implemented in a way that doesn't block.

Reply:
Updated this to only take the lock when destroying the swapchain.